### ISSUE TITLE
 '<' not supported between instances of 'int' and 'NoneType'

### ISSUE THREAD

**Issue by BainOuO** [PaddlePaddle/PaddleClas#1900]
欢迎您使用PaddleClas并反馈相关问题，非常感谢您对PaddleClas的贡献！
提出issue时，辛苦您提供以下信息，方便我们快速定位问题并及时有效地解决您的问题：

### 必要信息
** 如果您不能提供以下必要信息，可能会影响问题解决的速度 **
#### 1. PaddleClas版本以及PaddlePaddle版本
PaddlePaddle 2.1.0 
##### 2.1 原始代码问题
 '<' not supported between instances of 'int' and 'NoneType'
#### 3. 报错信息和log

2022-05-09 11:16:10 INFO: 
===========================================================
==        PaddleClas is powered by PaddlePaddle !        ==
===========================================================
==                                                       ==
==   For more info please go to the following website.   ==
==                                                       ==
==       https://github.com/PaddlePaddle/PaddleClas      ==
===========================================================

2022-05-09 11:16:10 INFO: ARCHITECTURE : 
2022-05-09 11:16:10 INFO:     name : MobileNetV3_large_x1_25
2022-05-09 11:16:10 INFO: ------------------------------------------------------------
2022-05-09 11:16:10 INFO: LEARNING_RATE : 
2022-05-09 11:16:10 INFO:     function : CosineWarmup
2022-05-09 11:16:10 INFO:     params : 
2022-05-09 11:16:10 INFO:         lr : 0.65
2022-05-09 11:16:10 INFO:         warmup_epoch : 5
2022-05-09 11:16:10 INFO: ------------------------------------------------------------
2022-05-09 11:16:10 INFO: OPTIMIZER : 
2022-05-09 11:16:10 INFO:     function : Momentum
2022-05-09 11:16:10 INFO:     params : 
2022-05-09 11:16:10 INFO:         momentum : 0.9
2022-05-09 11:16:10 INFO:     regularizer : 
2022-05-09 11:16:10 INFO:         factor : 4e-05
2022-05-09 11:16:10 INFO:         function : L2
2022-05-09 11:16:10 INFO: ------------------------------------------------------------
2022-05-09 11:16:10 INFO: TRAIN : 
2022-05-09 11:16:10 INFO:     batch_size : 128
2022-05-09 11:16:10 INFO:     data_dir : ./dataset/tea/
2022-05-09 11:16:10 INFO:     file_list : ./dataset/tea/train_list.txt
2022-05-09 11:16:10 INFO:     num_workers : 1
2022-05-09 11:16:10 INFO:     shuffle_seed : 0
2022-05-09 11:16:10 INFO:     transforms : 
2022-05-09 11:16:10 INFO:         DecodeImage : 
2022-05-09 11:16:10 INFO:             channel_first : False
2022-05-09 11:16:10 INFO:             to_rgb : True
2022-05-09 11:16:10 INFO:         RandCropImage : 
2022-05-09 11:16:10 INFO:             size : 224
2022-05-09 11:16:10 INFO:         RandFlipImage : 
2022-05-09 11:16:10 INFO:             flip_code : 1
2022-05-09 11:16:10 INFO:         NormalizeImage : 
2022-05-09 11:16:10 INFO:             mean : [0.485, 0.456, 0.406]
2022-05-09 11:16:10 INFO:             order : 
2022-05-09 11:16:10 INFO:             scale : 1./255.
2022-05-09 11:16:10 INFO:             std : [0.229, 0.224, 0.225]
2022-05-09 11:16:10 INFO:         ToCHWImage : None
2022-05-09 11:16:10 INFO: ------------------------------------------------------------
2022-05-09 11:16:10 INFO: VALID : 
2022-05-09 11:16:10 INFO:     batch_size : 128
2022-05-09 11:16:10 INFO:     data_dir : ./dataset/tea/
2022-05-09 11:16:10 INFO:     file_list : ./dataset/tea/val_list.txt
2022-05-09 11:16:10 INFO:     num_workers : 1
2022-05-09 11:16:10 INFO:     shuffle_seed : 0
2022-05-09 11:16:10 INFO:     transforms : 
2022-05-09 11:16:10 INFO:         DecodeImage : 
2022-05-09 11:16:10 INFO:             channel_first : False
2022-05-09 11:16:10 INFO:             to_rgb : True
2022-05-09 11:16:10 INFO:         ResizeImage : 
2022-05-09 11:16:10 INFO:             resize_short : 256
2022-05-09 11:16:10 INFO:         CropImage : 
2022-05-09 11:16:10 INFO:             size : 224
2022-05-09 11:16:10 INFO:         NormalizeImage : 
2022-05-09 11:16:10 INFO:             mean : [0.485, 0.456, 0.406]
2022-05-09 11:16:10 INFO:             order : 
2022-05-09 11:16:10 INFO:             scale : 1.0/255.0
2022-05-09 11:16:10 INFO:             std : [0.229, 0.224, 0.225]
2022-05-09 11:16:10 INFO:         ToCHWImage : None
2022-05-09 11:16:10 INFO: ------------------------------------------------------------
2022-05-09 11:16:10 INFO: classes_num : 5
2022-05-09 11:16:10 INFO: epochs : 100
2022-05-09 11:16:10 INFO: image_shape : [3, 224, 224]
2022-05-09 11:16:10 INFO: ls_epsilon : 0.1
2022-05-09 11:16:10 INFO: mode : train
2022-05-09 11:16:10 INFO: model_save_dir : ./output/
2022-05-09 11:16:10 INFO: pretrained_model : 
2022-05-09 11:16:10 INFO: save_interval : 1
2022-05-09 11:16:10 INFO: topk : None
2022-05-09 11:16:10 INFO: total_images : 27662
2022-05-09 11:16:10 INFO: valid_interval : 1
2022-05-09 11:16:10 INFO: validate : True
W0509 11:16:10.537891   118 device_context.cc:447] Please NOTE: device: 0, GPU Compute Capability: 7.0, Driver API Version: 11.2, Runtime API Version: 10.1
W0509 11:16:10.544189   118 device_context.cc:465] device: 0, cuDNN Version: 7.6.
2022-05-09 11:16:16 ERROR: '<' not supported between instances of 'int' and 'NoneType'

**Comment by cuicheng01** [comment]
您好，可能是数据集list的问题，请确认下数据集的格式是“xxx.jpg label”
